### PR TITLE
add all regions to default config in O365 feed

### PR DESCRIPTION
**Pull request by yuvalbenshalom**
## Status
- [ ] In Progress
- [ ] Ready
- [x] In Hold - pending customer verification

## Related Issues
fixes: https://github.com/demisto/etc/issues/41380

## Description
The region "Worldwide" in the feed configuration does not include the other regions, as the name may falsely suggest.
since this feed is generally used to replace minemeld "O365  any" prototype that includes all the regions, the default configuration should include all of them.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedOffice365/ReleaseNotes/1_1_11.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Office 365 Feed
+- Changed the feed configuration to include all the regions by default.
```
Comment (bakatzir) [inline Packs/FeedOffice365/ReleaseNotes/1_1_11.md:3]:
```suggestion
- Changed the feed default configuration to include all of the regions.
```

**Review by bakatzir — APPROVED** [review]

**Review by glicht — COMMENTED** [review]
How about adding a new value of "All" which when set the code will use all regions? I think it can be a clearer experience.

File: Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml [hunk 2]
```diff
@@ -8,10 +8,11 @@ configuration:
   defaultvalue: 'true'
   required: false
   type: 8
-- defaultvalue: Worldwide
+- defaultvalue: All
```
Comment (bakatzir) [inline Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.yml:11]:
```suggestion
- defaultvalue: All
  additionalinfo: "The Office 365 regions to pull indicators from. 'All' represents all of the available regions."
```

**Review by bakatzir — APPROVED** [review]
nice. see my comment regarding addiitonalinfo. I think it might be helpful.

**Pull request merged**